test/clj_git/test/core_test.clj

```diff
@@ -3,13 +3,32 @@
             [clojure.test :refer :all])
   (:require [clj-git.core :as git] :reload-all))
 
+(def es-src-dir (-> "clj_git"
+                    io/resource
+                    io/file
+                    .getParent
+                    (io/file "elasticsearch-src")
+                    .getCanonicalPath))
+
+(defn maybe-prep-repo [dir]
+  (if (.exists (io/file dir ".git"))
+    (do
+      (print "Elasticsearch source already cloned.  Pulling...")
+      (flush)
+      (git/git-pull (git/load-repo dir)))
+    (do
+      (print "Elasticsearch source missing.  Cloning...")
+      (flush)
+      (git/git-clone
+       dir
+       "git@github.com:elastic/elasticsearch.git")))
```

Comment:
Should this clone over HTTPS since it's a public repo and then you don't need to rely on the SSH environment being set up?